integration-tests/gatsby-pipeline/README.md

```diff
@@ -0,0 +1,30 @@
+# gatsby-pipeline
+
+It's a default gatsby starter to test our build & develop pipeline.
+
+For an overview of the project structure please refer to the [Gatsby documentation - Building with Components](https://www.gatsbyjs.org/docs/building-with-components/).
+
+## Install
```

Comment:
Maybe make these instructions relevant to this actual pipeline integration test?